tests/unittests/Foundation/NSNotificationQueueTests.mm

```diff
@@ -18,8 +18,10 @@
 #import <TestFramework.h>
 #import <Starboard/SmartTypes.h>
 #import <Helpers/TestHelpers.h>
+#import <Windows.h>
```

Comment:
There're three issues here:

* The tests don't compile on OS X
* The tests don't _work_ on OS X
* The tests can't be in WindowsOnly because of ... something? Something fun and compiler-related due to the fact that WO tests consume Foundation*Lib*